### DISCUSSION TITLE
Welcome to nuxtjs.org Discussions!

### DISCUSSION THREAD

**Discussion by pi0** [nuxt/website-v2#1521] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by febrihidayan** [comment]
are the nuxt 2 and nuxt 3 guides much different? i'm a little confused to start, maybe should follow the main language guide nuxt 3

**Comment by oritwoen** [comment]
What is the current password for V3 documentation?

https://v3.nuxtjs.org

**Reply by febrihidayan** [reply]
bisa coba ini https://preview.nuxtjs.org/

**Comment by rah-emil** [comment]
Can I help you with RU version?😇
